frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTSelectZone.js

```
@@ -0,0 +1,207 @@
+import { CardLabel, CardLabelError, Dropdown, FormStep, LabelFieldPair, RadioOrSelect } from "@egovernments/digit-ui-react-components";
+import _ from "lodash";
+import React, { useEffect, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useLocation } from "react-router-dom";
+import Timeline from "../components/TLTimeline";
+
+const PTSelectZone = ({ t, config, onSelect, userType, formData, setError, clearErrors, formState }) => {
+  const allCities = Digit.Hooks.pt.useTenants();
+  let tenantId = Digit.ULBService.getCurrentTenantId();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tenantId' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)